src/ObjectPool/src/IResettable.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.ObjectPool;
+
+/// <summary>
+/// Defines a method to reset an object to its initial state.
```

Comment:
```suggestion
/// Defines a method to reset an object to its initial state. This is used by <see cref="DefaultPooledObjectPolicy{T}"/> if implemented.
```

While others are free to call it, I think it's worth pointing out that this is the only thing that calls it by default.